dcos/package.py

```diff
@@ -1340,60 +1340,66 @@ def _data(self, path):
         full_path = os.path.join(self.path, path)
```

Comment:
There are other references to "version" that need to be changed to "revision" in this file.  e.g. "def package_json(version)"